provisioning/ubuntu-provision.sh

```diff
@@ -88,6 +88,10 @@ function setuser() {
   # Create agent folder
   mkdir -p "${userhome}/agent"
   chown -R jenkins:jenkins "${userhome}/agent"
+
+  # Ensure entrypoint script is executable
+  chown -R jenkins:jenkins /usr/local/bin/entrypoint.sh
+  chmod +x /usr/local/bin/entrypoint.sh
```

Comment:
```suggestion
  chmod a+x /usr/local/bin/entrypoint.sh
```

easier to make sure that any user could execute the script (the user running the agent **should** be `jenkins` but it can be changed at runtime)